src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'no-explicit-any' eslint rule disable comment, instead consider using a more specific type for 'data' parameter in the 'request' function. For example, you can use 'Record<string, unknown>' or create a custom type for your API data.